### DISCUSSION TITLE
Inquiry about API Access for Vehicle Make and Model Data

### DISCUSSION THREAD

**Discussion by P-seraph** [transitland/transitland#429] [Q&A] [answered]
we are in the process of sourcing reliable and comprehensive data to populate our platform with vehicle make and model information.

I am writing to inquire if your company offers an API that provides access to vehicle make and model data. Specifically, we are interested in obtaining an API connection that delivers identification information for various vehicle categories. It is important to note that we do not require parts information at this time, as we are focusing solely on make and model details.

Could you please provide information on the following:
-The availability of an API for vehicle make and model data.
-Subscription or license-based pricing models for accessing this API.
-Any limitations or requirements associated with using your API.
-Technical documentation or resources to better understand the integration process.

**Comment by drewda** [comment, marked as answer]
Transitland does not collect or provide any information about vehicles. Thank you.
